jit-transfer-obj/pom.xml

```diff
@@ -24,6 +24,14 @@
 			<groupId>com.fasterxml.jackson.core</groupId>
 			<artifactId>jackson-annotations</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
Hmm, TO's really shouldn't depend on anything in shared-kernel actually. What do you need in domain and infrastructure that causes you to put these dependencies?

Reply:
ValidVesselIMONumber and FacilityTypeCode/Address.

Reply:
Dammit, yeah.

Reply:
Removed infrastructure dependency.